lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -3667,6 +3667,28 @@ void SwiftASTContext::CacheModule(swift::ModuleDecl *module) {
   m_swift_module_cache.insert({ID, module});
 }
 
+void SwiftASTContext::MapModuleABINameToRealName(swift::ModuleDecl *module) {
+  if (module->getABIName() == module->getName())
+    return;
+
+  // Ignore _Concurrency, which is hardcoded in the compiler and should be
+  // lookup up using its ABI name "Swift"
+  if (module->getName().str() == swift::SWIFT_CONCURRENCY_NAME)
+    return;
+
+  // Also ignore modules with the special "Compiler" prefix.
+  if (module->getABIName().str().starts_with(
+          swift::SWIFT_MODULE_ABI_NAME_PREFIX))
+    return;
+
+  LOG_PRINTF(GetLog(LLDBLog::Types),
+             "Mapping module ABI name \"%s\" to its regular name \"%s\"",
+             module->getABIName().str().begin(),
```

Comment:
```suggestion
             module->getABIName().str().c_str(),
```
I am really not sure if these are guaranteed to be nullterminated!